tests/ndarrays/conftest.py

```diff
@@ -1,51 +1,43 @@
 # pylint: disable=missing-function-docstring, missing-module-docstring
-# pylint: disable=arguments-differ, inconsistent-return-statements, protected-access, abstract-method
 import subprocess
 import os
 import pathlib
 import sys
 import shutil
 import pytest
 
-NEEDS_FROM_PARENT = hasattr(pytest.Item, "from_parent")
-
-def pytest_collect_file(parent, path):
+def pytest_collect_file(parent, file_path):
     """
     A hook to collect test_*.c test files.
 
-    """
-    if path.ext == ".c" and path.basename.startswith("test_"):
-        if NEEDS_FROM_PARENT:
-            return CTestFile.from_parent(path=pathlib.Path(path), parent=parent)
-        return CTestFile(parent=parent, path=pathlib.Path(path))
-
-def pytest_collection_modifyitems(items):
-    """
-    a hook to modify the items before the tests for C unit test files.
+    A hook to collect test_*.c test files.
```

Comment:
duplicated line.

Reply:
Yes. I assumed that the documentation tests would be run on this file so I tried to write "extended descriptions". It seems it is not parsed though so I can remove them

Reply:
Fixed ea0fd4c2